src/app/map/page.jsx

```diff
@@ -1,13 +1,9 @@
 "use client";
 
-import dynamic from "next/dynamic";
 import { useEffect, useState } from "react";
+import { Map } from "@/components/Map/Map";
 
 // Remove SSR for MapComponent, Leaflet uses window requiring client-side rendering
-const MapComponent = dynamic(() => import("../../components/Map/Map"), {
```

Comment:
Why is this removed?

Reply:
It was causing runtime errors and we don't need SSR for this app at all IMO

Reply:
This actually removes the pre-rendering of the HTML page for NextJS that caused problems in leaflet, as it requires the "window" object from the client. What runtime error did you get?